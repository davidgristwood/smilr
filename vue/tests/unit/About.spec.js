import { shallowMount } from '@vue/test-utils'
import About from '@/components/About.vue'

describe('About.vue', () => {
  it('renders current app version', () => {

    const wrapper = shallowMount(About, {})
    expect(wrapper).toMatchSnapshot()
  })
})
