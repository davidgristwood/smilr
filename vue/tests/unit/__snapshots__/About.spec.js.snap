// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About.vue renders current app version 1`] = `
<div>
  <b-card border-variant="primary" header-bg-variant="primary" header-text-variant="white">
    <h1 slot="header">
      <fa icon="info-circle"></fa> &nbsp; About Smilr
    </h1>
    <h2>A microservices &amp; SPA reference / demonstration app</h2> <img src="" class="aboutface animated rubberBand infinite slow">
    <h5></h5>
    <hr> <b>Version: 2.9.1 (Vue.js)</b><br>
    Â© Ben Coleman, 2017 - 2019<br>
    Provided under the <a target="_blank" href="https://opensource.org/licenses/MIT">MIT License</a><br><br>
    <b-button size="lg" variant="primary" href="https://smilr.benco.io" target="_blank">
      Project details &amp; source code (GitHub)
    </b-button>
    <hr> <pre><b>API Endpoint:</b> /api
<b>Mode:</b> test
<b>Build Info:</b> No build info

<b>More Details:</b> <a href="/api/info" target="_blank">/api/info</a></pre>
  </b-card>
</div>
`;
